donor-portal/src/pages/home/styles.css

```diff
@@ -76,3 +80,28 @@
     flex-direction: column;
   }
 }
+
+ .current-donors{
+   margin: 2rem 2rem 1rem 2rem;
+  padding: 1.5rem;
+  align-items: center;
+}
+ 
+.current-donors h2 {
+  color: var(--elixir-color-red);
+  text-align: center;
+  padding: 1rem;
+  font-size: 1.75rem;
+}
+
+.current-donors .donor-logo img{
+ max-width: 150px;
+max-height: 150px;
+padding: 3rem 3rem 3rem 3rem;
+}
+.current-donors .logo-container {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+ justify-content: center;
+align-items: center;
+}
```

Comment:
Add a extra line here

Reply:
@anjula-sack It seems linter is not formatting CSS files. Could you please check it?